clients/src/main/java/org/apache/kafka/common/requests/DescribeGroupsResponse.java

```diff
@@ -129,9 +129,9 @@ public int throttleTimeMs() {
     @Override
     public Map<Errors, Integer> errorCounts() {
         Map<Errors, Integer> errorCounts = new HashMap<>();
-        data.groups().forEach(describedGroup -> {
+        for (DescribedGroup describedGroup : data.groups()) {
```

Comment:
It seems like the previous approach is actually more concise and as clear?

Reply:
Fair enough, I reverted those two changes. Would you like me to also change the places where for statements were already being used?

Reply:
If you would like to clean that up, sure.

Reply:
@ijuma done.